### DISCUSSION TITLE
WebSocket 질문 😎

### DISCUSSION THREAD

**Discussion by byulla** [re-prife/back-end#209] [Q&A] [answered]
이번에 졸업전시에서 작품 잘 봤습니다!
웹 소켓을 사용하셨다고 들었는데 혹시 사용하시면서 어려운 점은 없으셨나요?

**Comment by m04j00** [comment, marked as answer]
소켓 기능을 구현하는데 소요할 시간이 충분하지 못해 자바 소켓을 공부하고 구현하기에는 벅찬 감이 있었습니다.
그래서 자바 소켓보다 접근하기 쉬운 Node Socket.IO를 사용하여 구현하게 되었습니다.
